config/config.go

```diff
@@ -193,6 +193,7 @@ type GlobalConfig struct {
 	SlackURL        string `yaml:"slack_url"`
 	PagerdutyURL    string `yaml:"pagerduty_url"`
 	OpsGenieAPIHost string `yaml:"opsgenie_api_host"`
+	ExternalURL     string `yaml:"external_url"`
```

Comment:
Let's not change it until we figure out what the division is.

Having things that are easy to change in the config is doing things the wrong way around. It's things that need to be easy to change or that are too complex to represent in a flag that need to go in the config (and thus implicitly be easy to change).